pyqueue_celery_processor/test/test_client.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from unittest.mock import patch
+from pyqueue_celery_processor.client import Client
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

put a blank line between each group of imports.